src/ucp/tag/offload.c

```diff
@@ -21,28 +21,28 @@ int ucp_tag_offload_iface_activate(ucp_worker_iface_t *iface)
     ucp_worker_t *worker   = iface->worker;
     ucp_context_t *context = worker->context;
 
-    if (!worker->tm.offload.num_ifaces) {
+    if (!worker->tm.offload.activated) {
         ucs_assert(worker->tm.offload.thresh       == SIZE_MAX);
         ucs_assert(worker->tm.offload.zcopy_thresh == SIZE_MAX);
         ucs_assert(worker->tm.offload.iface        == NULL);
 
         worker->tm.offload.thresh       = context->config.ext.tm_thresh;
         worker->tm.offload.zcopy_thresh = context->config.ext.tm_max_bb_size;
+        worker->tm.offload.activated    = 1;
```

Comment:
can we check worker->tm.offload.iface == NULL instead of adding "activated" field?